### ISSUE TITLE
identify the git commit from which docs were pushed

### ISSUE THREAD

**Issue by dchelimsky** [cucumber-ltd/relish-gem#11]
The problem:

I've got docs in my git repo for a future version (so I don't want to publish them), but I've got improvements of the current docs that I want to push. The problem is that I don't know what git commit I was on the last time I pushed. I could go back to the tag of the last release, but I'm pretty sure I've made some enhancements since then.

The proposed solution:

`relish push` can store data in a `.relish-data` (or some such) file that includes the sha1 of git HEAD. Then I can simply roll back to that commit, branch off it, and update the docs from there.

I'm not at all married to this solution, so other ideas are more than welcome.

**Comment by dchelimsky** [comment]
One other idea would be that this data could be stored and displayed on http://relishapp.com/dashboard

**Comment by mattwynne** [comment]
Yeah, I've thought before that it would be useful to send this kind of metadata to the server. It would give us the option to build plugins to integrate with source control somehow.

**Comment by dchelimsky** [comment]
An alternative that would solve this problem for me would be the ability to push one file into an existing "deployment" (for lack of a better word).
